lms/djangoapps/commerce/api/v0/urls.py

```diff
@@ -7,7 +7,7 @@
 BASKET_URLS = patterns(
     '',
     url(r'^$', views.BasketsView.as_view(), name='create'),
-    url(r'^{}/order/$'.format(r'(?P<basket_id>[\w]+)'), views.BasketOrderView.as_view(), name='retrieve_order'),
+    url(r'^(?P<basket_id>[\w]+)/order/$', views.BasketOrderView.as_view(), name='retrieve_order'),
```

Comment:
Should this be marked as deprecated? It might also be worth creating a dev story to delete it, so we don't forget about it.